naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/GraphqlSchemaProvider.scala

```diff
@@ -73,7 +74,10 @@ class DefaultGraphqlSchemaProvider @Inject() (schemaProvider: SchemaProvider)
       }
     }.toMap
     try {
-      val builder = new SangriaGraphQlSchemaBuilder(latestSchema.resources, types)
+      val resources = latestSchema.resources.filter { resource =>
```

Comment:
i agree. i would recommend putting it inside https://github.com/coursera/naptime/blob/master/naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/SangriaGraphQlSchemaBuilder.scala#L97 for top-level resources.

and if there's a related resource that uses a forward relation, that should also not be added if there's not a multiget available.

in both of these cases, we should log the error to be able to report these problems better / handle them at a later time